cves/kernel/CVE-2021-27365.yml

```diff
@@ -380,8 +403,8 @@ vouch:
 
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of what your answer was.
-  answer:
-  note:
+  answer: true
+  note: The fix was signed off on by other developers at Oracle and RedHat.
```

Comment:
Might be beneficial to supply the names of the people who signed off on the work.